packages/ember-metal/lib/meta.js

```diff
@@ -33,7 +32,7 @@ if (DEBUG) {
 @module ember-metal
 */
 
-export const UNDEFINED = symbol('undefined');
```

Comment:
Landed in https://github.com/emberjs/ember.js/commit/9471527454b30b9a892d6d40f21735365b5a2897. If we're going to change this it should be with some analysis of what v8 is doing with this.